### PR TITLE
PSR-2 compatibility, support for PHP 7 return type

### PR DESCRIPTION
**Pull request by gabfl**
(re-made a cleaner pr)

Sample:
```
    /**
     * myMethod function.
     *
     * @access public
     * @param int $a (default: 1)
     * @param string $b (default: 'abc')
     * @param array $c (default: [])
     * @return array
     */
    public function myMethod (int $a = 1, string $b = 'abc', array $c = []): array
    {
        
    }
```

This branch:
 - Strips spaces at the right of the docblock since they are not PSR-2 compatible
 - Add support for return types like `: array`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,9 +16,9 @@
 info = dict(
     CodaAuthorString = 'Justin Hileman',
     CodaDescriptionString = 'Generate PHP Docblocks like magick!',
-    CFBundleVersion = '2.0.1',
+    CFBundleVersion = '2.0.2',
```
Comment (bobthecow) [inline setup.py]: Let's leave the bundle version the same and I'll bump it when I tag the release.

**Review by bobthecow — CHANGES_REQUESTED** [review]
Looks good, thanks! Just one small change :)

**Comment by gabfl** [comment]
Hi @bobthecow , its done!

**Comment by bobthecow** [comment]
thanks!

**Pull request merged**